parts/k8s/kuberneteswinagentresourcesvmas.t

```diff
@@ -260,7 +260,7 @@
     {{if UseAksExtension}}
     ,{
       "type": "Microsoft.Compute/virtualMachines/extensions",
-      "name": "[concat(variables('{{.Name}}VMNamePrefix'), copyIndex(variables('{{.Name}}Offset')), '/computeAksLinuxBilling')]",
+      "name": "[concat(variables('{{.Name}}VMNamePrefix'), copyIndex(variables('{{.Name}}Offset')), '/computeAksWindowsBilling')]",
```

Comment:
Not sure if the `computeAksWindowsBilling` extension exists from a billing perspective

Reply:
nevermind, this is the name of the extension in the template but the actual extension will be the same :)